deploy/crds/toolchain.dev.openshift.com_tiertemplates.yaml

```diff
@@ -0,0 +1,126 @@
+
```

Comment:
is it a new manifest, or maybe git did not detect that it's a file that was renamed?

Reply:
I noticed it too but it seem there are some minor changes in the file and this is why it's not detected as renaming but as two changes: new file + deletion of the old one.

Reply:
I saw that too when I was making the commit - as you can see, there is also deleted `toolchain_v1alpha1_tiertemplate_crd.yaml` file so GH just probably didn't recognize the change. :shrug: